### PR TITLE
add subscribe button to header and footer

### PR DESCRIPTION
**Pull request by catherinewu**

### CODE REVIEW

File: components/Footer.tsx [hunk 1]
```diff
@@ -68,6 +68,21 @@ const Footer = () => {
...
+                  clipRule="evenodd" 
+                />
+              </svg> */}
+              <svg className="h-6 w-6" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="envelope-open-text" class="svg-inline--fa fa-envelope-open-text fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="currentColor" d="M176 216h160c8.84 0 16-7.16 16-16v-16c0-8.84-7.16-16-16-16H176c-8.84 0-16 7.16-16 16v16c0 8.84 7.16 16 16 16zm-16 80c0 8.84 7.16 16 16 16h160c8.84 0 16-7.16 16-16v-16c0-8.84-7.16-16-16-16H176c-8.84 0-16 7.16-16 16v16zm96 121.13c-16.42 0-32.84-5.06-46.86-15.19L0 250.86V464c0 26.51 21.49 48 48 48h416c26.51 0 48-21.49 48-48V250.86L302.86 401.94c-14.02 10.12-30.44 15.19-46.86 15.19zm237.61-254.18c-8.85-6.94-17.24-13.47-29.61-22.81V96c0-26.51-21.49-48-48-48h-77.55c-3.04-2.2-5.87-4.26-9.04-6.56C312.6 29.17 279.2-.35 256 0c-23.2-.35-56.59 29.17-73.41 41.44-3.17 2.3-6 4.36-9.04 6.56H96c-26.51 0-48 21.49-48 48v44.14c-12.37 9.33-20.76 15.87-29.61 22.81A47.995 47.995 0 0 0 0 200.72v10.65l96 69.35V96h320v184.72l96-69.35v-10.65c0-14.74-6.78-28.67-18.39-37.77z"></path></svg>
```
Comment (yuhan) [inline components/Footer.tsx:84]: the build is failing bc you have have multiple attributes with the same name. merging the two `viewBox` attr and `className` attr (you have `className` and `class`) should fix it.

Comment (catherinewu) [reply components/Footer.tsx]: yup haha

Comment (catherinewu) [reply components/Footer.tsx]: im not sure how to "downsize" the path from 512 -> 24. Also, I think adding 'mr-6' to the class is needed to center the logos?

Comment (catherinewu) [reply components/Footer.tsx]: is the email icon reasonable for now? i looked for better ones but none jumped out. also the stackoverflow icon looks small in comparison

Comment (yuhan) [reply components/Footer.tsx]: yep i think adding `mr-6` to the last item is a good call! thanks for fixing that

Comment (yuhan) [reply components/Footer.tsx]: lg2m! the stackoverflow icon size the same as others but the svg path draws a bit inside the 24x24 box while the github and slack icon paths are a bit closer to the box boundary... let's leave it as it is until we find a better colorful svg for it :)

**Review by yuhan — APPROVED** [review]

File: components/Footer.tsx (hunk 1 above)
Comment (yuhan) [reply components/Footer.tsx]: btw make sure you clean up the commented code before merging :)

Comment (catherinewu) [reply components/Footer.tsx]: does the "commented code" refer to `{/* https://fontawesome.com/license */}`? When I downloaded the svg, fontawesome asked for the license to be included wherever the svg is used unless we have the pro license (which I don't think we do?)

Comment (catherinewu) [reply components/Footer.tsx]: i can clarify that in the comment and also add the comment to the footer too

**Pull request merged**